### DISCUSSION TITLE
Chargin Cutoff

### DISCUSSION THREAD

**Discussion by danielducu** [PricelessToolkit/MailBoxGuard#44] [Q&A]
Hello, just finished mounting this board and everything works ok but have doubts about the charging circuit ... right now the CH led is lit and the battery has been charged to about 4.18v... at what point is the voltage cutoff ... im worried something is wrong and it will overcharge?

**Comment by danielducu** [comment]
Never mind man, it stopped at 4.20v ...

**Reply by PricelessToolkit** [reply]
ok :)
